chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -1,40 +1,116 @@
 from __future__ import division
+
 import numpy as np
+import six
+
+from chainer import cuda
 
 
 def _fast_hist(label_true, label_pred, n_class):
-    """Construct histogram for label evaluation.
+    # Construct histogram for label evaluation.
 
-    Returns
-        numpy.ndarray of shape (n_class, n_class)
-    """
     mask = (label_true >= 0) & (label_true < n_class)
+    # an array of shape (n_class, n_class)
     hist = np.bincount(
         n_class * label_true[mask].astype(int) +
         label_pred[mask], minlength=n_class**2).reshape(n_class, n_class)
     return hist
 
 
-def label_accuracy_score(label_true, label_pred, n_class):
-    """Returns accuracy score evaluation result.
+def eval_semantic_segmentation(label_pred, label_true, n_class):
```

Comment:
I think we should make this consistent with other codes as Hakuyume pointed out.
Although `label_true` or `true_label` sounds natural in the case of evaluation code, in other part of the code, `gt_label` sounds better.
Since we have to pick one, I would choose `gt_label`.